site/content/blogs/release-v131.en.md

```diff
@@ -20,6 +20,28 @@ Copilot v1.31 brings big enhancements to help you develop more flexibly and effi
 
 - **NLB enhancements**: You can now add security groups to Copilot-managed NLBs. NLBs also support the UDP protocol.
 - **Better task failure logs**: Copilot will show more descriptive information during deployments when tasks fail, allowing better troubleshooting.
+
+You can view the ECS task stopped reasons for the two most recent tasks during deployments using `copilot [noun] deploy`.
```

Comment:
lol we should add these section below https://github.com/aws/copilot-cli/pull/5346/files#diff-c933228c363a1118c64c52bd0c84346dbac20d74a73c370c3e0df06394003a10R59

Reply:
i guess it only makes sense for ecs service deploy? `copilot [noun] deploy` might be a little confusing. Maybe we just say "... during deployments of your LBWS/BS/Worker svcs"? Also, it would be great to mention the reason why we wanted to prioritize this feature at the very beginning of the section (more context for readers)?

Reply:
> Also, it would be great to mention the reason why we wanted to prioritize this feature at the very beginning of the section.

what other info do you want me to add 🤔

Reply:
Just some background will be good. Like without this feature what inconvenience they had before